drydock/templates/drydock/k8s/drydock-jobs/cms.yml

```
@@ -1,9 +1,9 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: cms-job
+  name: drydock-cms-job
   labels:
-    app.kubernetes.io/component: job
+    shipyard.io/component: job
```

```suggestion
    drydock.io/component: job
```
Should we keep uniformity in the label naming? It applies for the rest of jobs